gradle.properties

```diff
@@ -4,10 +4,10 @@
 # idea_version=211.6693.111
 # scala_plugin_version=2021.1.21
 
+pluginVersion=1.17.0
```

Comment:
ideally we would push a tag like v1.18.0 and it would publish with a version according to the tag. Current solution is also ok I guess

Reply:
I though about that and I can do it this way, however it will require a consistent tagging scheme and I'm not so sure it's the best solution. The plugin version in the properties should always be up to date for bleeding edge releases anyway. 
If you really want me to rework it though, then sure, I'll do

Reply:
As I said current solution is also ok.
Just to discuss, I am not sure how it worked before, we seem to have consistent tagging and bleeding edge versions. In sbt there is a plugin that also correctly assigns version, I think based on latest release tag (if you are on tag it is a release, else it is a snapshot).